docker-entrypoint.sh

```diff
@@ -205,6 +210,36 @@ _main() {
   fi
 
   _replace_conf_from_env
+
+  BACKUP_INIT_FOLDER="/docker-entrypoint-initdb.d"
+
+  if [[ $(ls -1 $BACKUP_INIT_FOLDER | wc -l) -eq 4 ]]; then
```

Comment:
What's the idea of comparing with 4 here? Please add a comment.

Reply:
It allows to check if the user mounted the backup path correctly (It's possible that the user will mount the folder instead backup's files)

Reply:
Why 4? Looks like an error-prone approach that we'll have to deal with in the future.